manifests/deployment.yaml

```diff
@@ -2,39 +2,58 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: kepler-model-server
+  namespace: monitoring
   labels:
-    app: sustainable-computing.io
+    app.kubernetes.io/component: model-server
+    app.kubernetes.io/name: kepler-model-server
 spec:
   replicas: 3
   selector:
     matchLabels:
-      app: sustainable-computing.io
+      app.kubernetes.io/component: model-server
+      app.kubernetes.io/name: kepler-model-server
   template:
     metadata:
       labels:
-        app: sustainable-computing.io
+        app.kubernetes.io/component: kepler-model-server
+        app.kubernetes.io/name: kepler-model-server
     spec:
+      volumes:
+      - name: models
+        hostPath:
+          path: /tmp/kepler-model-server
       containers:
       - name: kepler-model-server
         image: quay.io/sustainable_computing_io/kepler_model_server_base:latest
         ports:
         - containerPort: 8100
-...
-
-apiVersion: batch/v1
-kind: CronJob
-metadata:
-  name: kepler-model-trainer
-spec:
-  schedule: "* * * * *"
-  jobTemplate:
-    spec:
-      containers:
+        volumeMounts:
+          - name: models
+            mountPath: /tmp/kepler-model-server/models
```

Comment:
set the mountPath to the model path defined [here](https://github.com/sustainable-computing-io/kepler-model-server/blob/main/server/kepler_model_trainer.py#L9), ie. under the `current work dir/models`